ee/clickhouse/queries/test/test_cohort_query.py

```diff
@@ -3218,6 +3218,167 @@ def test_unwrap_multiple_levels(self):
 
         self.assertCountEqual([p4.uuid], [r[0] for r in res])
 
+    def test_performed_event_regularly_years_interval(self):
+        # Person 1: Matches the condition - has 2 "Application Opened" events
+        p1 = _create_person(
+            team_id=self.team.pk,
+            distinct_ids=["p1"],
+            properties={"name": "test1", "email": "test1@posthog.com"},
+        )
+        _create_event(
+            team=self.team,
+            event="Application Opened",
+            properties={},
+            distinct_id="p1",
+            timestamp=datetime.now() - timedelta(days=2),
+        )
+        _create_event(
+            team=self.team,
+            event="Application Opened",
+            properties={},
+            distinct_id="p1",
+            timestamp=datetime.now() - timedelta(days=1),
+        )
+
+        # Person 2: Doesn't match - has only 1 "Application Opened" event
```

Comment:
Would it make sense to add a case where the person has 2 "Application Opened" events, but one is outside of the time interval, and therefore doesn't match?